internal/operators/mce/config.go

```diff
@@ -13,6 +36,34 @@ const (
 	SNOMinimumCpu    int64 = 8
 )
 
-type Config struct {
-	MceMinOpenshiftVersion string `envconfig:"MCE_MIN_OPENSHIFT_VERSION" default:"4.10.0"`
+func ParseMCEConfig(config Config) (*ParsedConfig, error) {
+	var parsedMatrix []OCPMCEReleaseMatrix
+	if err := json.Unmarshal([]byte(config.OcpMceReleaseMatrix), &parsedMatrix); err != nil {
+		return nil, fmt.Errorf("failed to unmarshal ocp mce release matrix: %w", err)
+	}
+
+	return &ParsedConfig{
+		MceMinOpenshiftVersion: config.MceMinOpenshiftVersion,
```

Comment:
what is the usage of this variable?

Reply:
if it's used, it's fine to leave it as a `const`, not need to centralize everything in one place

Reply:
I see this approach in other operators as well, makes sense as it will make it easier to configure it differently in case it will be changed no ?